src/execution_plan/ops/op_update.c

```diff
@@ -211,7 +211,7 @@ static Record UpdateConsume(OpBase *opBase) {
 	if(!QueryCtx_LockForCommit()) return NULL;
 	_CommitUpdates(op);
 	// Release lock.
-	QueryCtx_UnlockCommit();
+	QueryCtx_UnlockCommit((OpBase *)op);
```

Comment:
Here and in other op files, the casting is unnecessary - you can just use `opBase`.

Reply:
done